### PR TITLE
Article page related stories

### PR DESCRIPTION
**Pull request by joshdarby**
## Changes

This pull request makes the following changes:

- Adds in a new Workday Related Stories widget, copied over from Largo Related Stories but also includes each posts top term. 
- Styles the new Workday Related Stories widget on posts.

Screen Shot 2019-08-06 at 9 09 17 AM

## Why

For #10 

## Testing/Questions

Features that this PR affects:

- Individual posts related stories area

Questions that need to be answered before merging:

- [x] Does it look ok?
- [x] Are the image sizes ok?
- [x] I set the sentence trim to 2, but have had to manually edit some posts since some don't have spaces after periods which is causing more than 2 sentences to be shown. Do we want to worry about that now or later?

Steps to test this PR:

1. Add `Workday Related Posts` widget to `Article Bottom` widget area.
2. Set number of posts to display at 3.
3. View article related stories section at multiple sizes.

### CODE REVIEW

**Comment by benlk** [comment]
I would've thought that this widget used the same code as the Recent Posts widget. Opened a Largo issue to fix that: https://github.com/INN/largo/issues/1769

**Comment by benlk** [comment]
> I set the sentence trim to 2, but have had to manually edit some posts since some don't have spaces after periods which is causing more than 2 sentences to be shown. Do we want to worry about that now or later?

No; it's on them to set a custom excerpt.

File: wp-content/themes/workday/inc/widgets/workday-related-posts.php [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+				<?php if ( isset( $instance['show_byline'] ) && $instance['show_byline'] ) { ?>
+					<h5 class="byline">
+						<span class="by-author"><?php largo_byline( true, false, get_the_ID() ); ?></span>
+					</h5>
+				<?php } ?>
+
+				<?php // post excerpt/summary
+				$excerpt = largo_excerpt(get_the_ID(), 2, false, '', true);
+
+				echo '</li>';
```
Comment (benlk) [inline wp-content/themes/workday/inc/widgets/workday-related-posts.php]: Would it make sense to replace lines 62-77 here with a call to use `partials/widget-content.php`, like in the related posts widget? https://github.com/INN/largo/blob/c41d8519370b48a6e53ce36b3b516d3e54e39d66/inc/widgets/largo-related-posts.php#L48-L66

(see https://github.com/INN/largo/issues/1769)

**Comment by benlk** [comment]
Image sizes look fine; collapses well on mobile. The comment in the code isn't a must-do; it's just something to think about for the future.

:shipit:

File: wp-content/themes/workday/inc/widgets/workday-related-posts.php (hunk 1 above)
Comment (joshdarby) [reply wp-content/themes/workday/inc/widgets/workday-related-posts.php]: Do you think it'd be better to do it now, or wait until we redo the Largo one and then just use that one once it's updated?

Comment (benlk) [reply wp-content/themes/workday/inc/widgets/workday-related-posts.php]: If they switched to the largo widget in the future, they'd need to update these styles to refer to that widget. Might be better to let this lie until we update their Largo for that future version of Largo.

**Pull request merged**